### PR TITLE
Update version of Geth used in CI

### PR DESCRIPTION
**Pull request by eggplantzzz**
This updates the version of Geth used by the CI as web3 should be compatible with the stable version now.

### CODE REVIEW

**Comment by eggplantzzz** [comment]
Without further changes it looks like the most recent version of Geth that Truffle plays well with is 1.10.3.

**Review by cds-amal — APPROVED** [review]
Nice @eggplantzzz! I love the TIL moments reviewing your PRs.

File: packages/truffle/test/scenarios/migrations/errors.js [hunk 1]
```diff
@@ -131,22 +131,23 @@ describe("migration errors", function() {
     }
   });
 
-  it("should error on insufficient funds correctly [ @geth ]", async function() {
+  it("should error on insufficient funds correctly [ @geth ]", async function () {
     this.timeout(70000);
 
     try {
-      await CommandRunner.run("migrate -f 6", config);
+      console.log("about to run a migration");
```
Comment (cds-amal) [inline packages/truffle/test/scenarios/migrations/errors.js:138]: I already approved, and just noticed this rogue console.log

File: packages/truffle/test/scenarios/migrations/errors.js [hunk 2]
```diff
@@ -7,17 +7,17 @@ const Reporter = require("../reporter");
...
 const Web3 = require("web3");
 
-describe("migration errors", function() {
+describe("migration errors", function () {
   let config;
   let web3;
   let networkId;
   const project = path.join(__dirname, "../../sources/migrations/error");
-  const logger = new MemoryLogger();
+  const logger = new MemoryLogger(true);
```
[the remark this answers is not in the document]
Comment (eggplantzzz) [reply packages/truffle/test/scenarios/migrations/errors.js:15]: I learned about this after I implemented this feature alongside it. I finished and then realized that's what `pipe` was for haha.

File: packages/truffle/test/scenarios/migrations/errors.js [hunk 3]
```diff
@@ -131,22 +131,23 @@ describe("migration errors", function() {
...
     try {
-      await CommandRunner.run("migrate -f 6", config);
+      console.log("about to run a migration");
+      await CommandRunner.run("migrate -f 10", config);
       assert(false, "This should have thrown.");
     } catch (_error) {
       const output = logger.contents();
       console.log(output);
-      assert(output.includes("6_migrations_funds.js"));
+      assert(output.includes("10_migrations_funds_geth.js"));
```
[the remark this answers is not in the document]
Comment (eggplantzzz) [reply packages/truffle/test/scenarios/migrations/errors.js:144]: Geth didn't like setting `gasPrice` lower than the `baseFeePerGas`. These needed to be separated for now until Ganache returns `baseFeePerGas` in its response for `getBlock`.

File: packages/truffle/test/sources/migrations/error/migrations/10_migrations_funds_geth.js [hunk 4]
```diff
@@ -0,0 +1,18 @@
+const Example = artifacts.require("Example");
+
+module.exports = async function (deployer, network, accounts) {
+  const emptyAccount = accounts[7];
+  let balance = await web3.eth.getBalance(emptyAccount);
+  const { baseFeePerGas } = await web3.eth.getBlock("latest");
+  // 21,000 gas to send ether
```
[the remark this answers is not in the document]
Comment (eggplantzzz) [reply packages/truffle/test/sources/migrations/error/migrations/10_migrations_funds_geth.js:7]: Is it not clear from `emptyAccount`? :)

File: packages/truffle/test/scenarios/migrations/errors.js (hunk 1 above)
Comment (eggplantzzz) [reply packages/truffle/test/scenarios/migrations/errors.js:138]: Oh whoops! Good catch!

**Pull request merged**